### ISSUE TITLE
Add italian language to nsis script for the installer - changes to nsis script

### ISSUE THREAD

**Issue by bovirus** [lelegard/qtlinguist-installers#7]
Hi.

Can i help you to localize the nsis qtlinguist script for the installer adding Italian language?

Where can I find the text strings used in the installer?

I got this info about nsis script

!include "MUI2.nsh"
!include "WinMessages.nsh"

WEhere can I find it?

I propose theese changes in nsis

OutFile "${RootDir}\installers\QtLinguist-${ProductVersion}**_-Installer_**.exe" (add -Installer)

InstallDir "$PROGRAMFILES\QtLinguist" (without space between Qt and Linguist)

I poprose also to change the standaone zip file for Windows to

QTLinguist-5.x.x**_-Standalone_**.zip (-Standalone after version number)

Thanks.

**Comment by lelegard** [comment]
Fixed in commit 8e9cd761073827a7200e9c2a83e48da79ae61ee5

I added not only Italian but all languages which are supported by NSIS.

Honestly, I did not do it in the first place because Qt Linguist is a tool for translators, people who speak English (most of the time) and who will translate applications from an original language (mostly English) to their own native language. So, internationalising a tool for translators is useless. It is maybe the only tool in the world which does not need translation :) 

Anyway, it way easy to do...
